src/dxvk/dxvk_graphics.cpp

```diff
@@ -396,14 +403,19 @@ namespace dxvk {
       dyInfo.pDynamicStates     = &dynamicState;
     }
 
+    VkPipelineCreateFlags flags = VK_PIPELINE_CREATE_LIBRARY_BIT_KHR;
+    if (device->features().extAttachmentFeedbackLoopLayout.attachmentFeedbackLoopLayout)
+      flags |= VK_PIPELINE_CREATE_COLOR_ATTACHMENT_FEEDBACK_LOOP_BIT_EXT
+            |  VK_PIPELINE_CREATE_DEPTH_STENCIL_ATTACHMENT_FEEDBACK_LOOP_BIT_EXT;
```

Comment:
We know the exact bits here because they are stored in `DxvkGraphicsPipelineFragmentOutputState`, so we should check those rather than unconditionally enabling the flags.

This would have been reasonable for fragment *shader* pipelines, but we don't need to pass the flags there.